plugins/org.eclipse.elk.alg.common/META-INF/MANIFEST.MF

```diff
@@ -16,6 +16,7 @@ Export-Package: org.eclipse.elk.alg.common,
  org.eclipse.elk.alg.common.networksimplex,
  org.eclipse.elk.alg.common.nodespacing,
  org.eclipse.elk.alg.common.nodespacing.cellsystem,
+ org.eclipse.elk.alg.common.nodespacing.internal.algorithm;x-friends:="org.eclipse.elk.alg.layered",
```

Comment:
It's been a while .. is the x-friends part necessary?

Reply:
No. I'll move methods from org.eclipse.elk.alg.common.nodespacing.internal.algorithm.NodeLabelAndSizeUtilities to org.eclipse.elk.core.util.ElkUtil.